src/main/java/com/dev/cinema/model/dto/CinemaHallDtoRequest.java

```diff
@@ -1,7 +1,12 @@
 package com.dev.cinema.model.dto;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PositiveOrZero;
```

Comment:
use `javax.validation.constraints` - not jakarta
for all RequestDtos

Reply:
Searched everywhere and changed.